onnxscript/backend/onnx_export.py

```diff
@@ -221,7 +270,9 @@ def make_opset_name(self, domain, version):
         return f"{self._rename_domain(domain)}{version}"
 
     def _python_make_node_name(self, domain, version, name, node=False):
-        name = _rename_variable(name)
+        name = _rename_variable(
+            name
+        )  # TODO: Is this a typo? Is it supposed to be self._rename_variable(name)?
```

Comment:
Need to fix this?

Reply:
I added that question ... I am not sure, need to look at it closely. I have another followup PR with other cleanup and will then get to this.